packages/kbn-shared-ux-components/src/page_template/solution_nav/solution_nav.test.tsx

```diff
@@ -59,6 +63,47 @@ const items: KibanaPageTemplateSolutionNavProps['items'] = [
 ];
 
 describe('KibanaPageTemplateSolutionNav', () => {
+  describe('isOpenOnMobile', () => {
+    test('defaults to false', () => {
```

Comment:
Yes, actually I moved those test cases from EuiSideNav, and now I see it makes no sense to test that parameters since they are just passed to EuiSideNav, there's no difference in our component, and they are already tested in EUI. I removed those duplications and I added a test for the `children` prop, checking the component instead of doing a snapshot.